docs/tutorials/e2e/boost/consumeData.md

```diff
@@ -187,21 +187,29 @@ The response shows her that she has a `STARTED` transfer with the type `CONSUMER
 Note, that a consumer pull transfer will **not** advance to the `COMPLETED` state, but instead will stay in state `STARTED`.
 This means, that Alice can now request the data using the information received by her backend.
 
+:::info
+
+If you used webhook.site as your backend system, you can now go to [webhook.site](https://webhook.site/) and see the received token. Please skip the next command if you didn't use edr before. Jump directly to the next section.
+
+:::
+
 ```shell
 curl --location 'http://localhost/alice/management/edrs?assetId=3' \
 --header 'X-Api-Key: password' | jq
 ```
```

Comment:
I would basically drop this. There is (as far as I know) no reference to the EDR API at all in this tutorial.

Reply:
deleted, thx @florianrusch-zf